src/site/resources/index.html

```diff
@@ -0,0 +1,160 @@
+                        <li>
+                            <a href="https://github.com/GoogleCloudPlatform/gcloud-java/issues" title="gcloud-java issues on Github" class="ext-link">
+                                <img src="img/icon-link-github.svg" alt="Github icon" />
+                                Issues
+                            </a>
+                        </li>
+                        <li>
+                            <a href="http://stackoverflow.com/questions/tagged/gcloud-java" title="gcloud on StackOverflow" class="ext-link">
+                                <img src="img/icon-link-stackoverflow.svg" alt="StackOverflow icon" />
+				StackOverflow
+                            </a>
+                        </li>
+                        <li>
+                            <a href="./project-info.html" title="Java package manager" class="ext-link">
+                                 <img src="img/icon-link-package-manager.svg" alt="Package Manager icon" />
+                                Package Information
+                            </a>
+                        </li>
+                    </ul>
+                </div><!-- end of .container -->
+            </section><!-- end of .featuring -->
+
+            <section class="block about">
+                <div class="container clearfix">
+                  <h3 class="block-title">What is it?</h3>
+
+                  <p><code>gcloud</code> is a client library for accessing Google
+		    Cloud Platform services that significantly reduces the boilerplate
+		    code you have to write. The library provides high-level API
+		    abstractions so they're easier to understand. It embraces
+		    idioms of Java, works well with the standard library, and
+		    integrates better with your codebase.
+		    All this means you spend more time creating code that matters
+		    to you.</p>
+		  
+		  <p><code>gcloud</code> is configured to access Google Cloud Platform
+		    services and authorize (OAuth 2.0) automatically on your behalf.
+		    With a quick install and a private key, you are up and ready
+		    to go. Better yet, if you are running on a Google Compute Engine
+		    instance, the installation is enough!</p>
+		  
+                  <h4>Retrieve Datastore Entries</h4>
+		  
+                  <div hljs="" language="js">
+		    <pre><code class="hljs java"><span class="hljs-keyword">import</span> com.google.gcloud.datastore.Datastore;
+<span class="hljs-keyword">import</span> com.google.gcloud.datastore.DatastoreFactory;
+<span class="hljs-keyword">import</span> com.google.gcloud.datastore.DatastoreOptions;
+<span class="hljs-keyword">import</span> com.google.gcloud.datastore.Entity;
+<span class="hljs-keyword">import</span> com.google.gcloud.datastore.Key;
+<span class="hljs-keyword">import</span> com.google.gcloud.datastore.KeyFactory;
+
+DatastoreOptions options = DatastoreOptions.builder().projectId(<span class="hljs-string">PROJECT_ID</span>).build();
```

Comment:
I see, @ajkannan if you think adding the tabbed view is simple I would say lets do it now otherwise lets create an issue for it (maybe in the case of 'elsewhere' we could add a comment
suggesting `gcloud login` could also be used for authentication [and selecting projectId]).

@jgeewax looks like in gcloud-node you still require project-id on CE or is that doc not up-to-date? Also, I wonder why we only mention CE and not AE?